backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/log/SonarLintLogger.java

```diff
@@ -144,8 +144,11 @@ private void doLogExtractingThrowable(Level level, String msg, Object[] argArray
     logOutput.log(tuple.getMessage(), level, tuple.getThrowable());
   }
 
-  public static String singlePlural(int count, String singular, String plural) {
-    return count == 1 ? singular : plural;
+  /**
+   * Append an 's' at the end of the word
+   */
+  public static String singlePlural(int count, String singular) {
```

Comment:
Note: this won't work with all words - e.g. `branch`/`branches`.

Reply:
This is expected, following a suggestion from Damien, we decided to use this simple implementation because all of the current usages in SLCORE were just appending a 's', hence the  javadoc